cmd/webhook/main.go

```diff
@@ -19,7 +19,7 @@ package main
 import (
 	"flag"
 
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
```

Comment:
There may also be some differences in how we iniitalize flags between glog and klog. See https://github.com/kubernetes-csi/external-provisioner/pull/225/files for an example.

Reply:
Searched flag.CommandLine in the this package nothing came up. Do we need to do this proactively?

Reply:
Double checked that we used commanline flags in  setup_e2e_test.go but it is already doing klog.InitFlags(flag.CommandLine).

Reply:
I think we need to add `klog.InitFlags()` here: https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/cmd/main.go#L57

Reply:
Added.